src/olympia/addons/tests/test_views.py

```diff
@@ -2255,8 +2256,10 @@ def test_get_queryset_excludes(self):
             key.endswith('.raw') for key in source_keys
         )
 
-    def perform_search(self, url, data=None, expected_status=200, **headers):
-        with self.assertNumQueries(0):
+    def perform_search(
+            self, url, data=None, expected_status=200, expected_queries=0,
+            **headers):
```

Comment:
nit: this isn't the style we usually write overflowing function definitions in.  Elsewhere we'd write it as:
```python
def perform_search(self, url, data=None, expected_status=200,
                   expected_queries=0, **headers):
    with self.assertNumQueries(expected_queries):
````